app/assets/v2/js/shared.js

```diff
@@ -705,10 +705,12 @@ var currentNetwork = function(network) {
           $('#current-network').text(gettext('Metamask Locked'));
           $('#navbar-network-banner').html(info);
         } else {
-          info = gettext('Metamask not connected. ') +
-            '<button id="metamask_connect" onclick="approve_metamask()">Click here to connect to metamask</button>';
-          $('#current-network').text(gettext('Metamask Not Connected'));
-          $('#navbar-network-banner').html(info);
+          if (window.ethereum._metamask) {
```

Comment:
can we make this into an `else if` in `L107` ?

Reply:
cf5de1e0c91ed3befd9c8967589360048b7cba4c